libs/brackets-server/embedded-ext/wasm-debug/main.js

```diff
@@ -40,19 +42,56 @@ define(function (require, exports, module) {
             return;
         }
 
-        const projectId = PreferencesManager.getViewState("projectId");
-        _nodeDomain.exec("wasm2wast", projectId, brackets.app.convertRelativePath(item.fullPath)).done(() => {
-            ProjectManager.refreshFileTree();
-        }).fail((err) => {
-            showErrorDialog(ExtensionStrings.ERROR_TRANSLATE_WASM);
-            console.log(err);
+        const basename = FileUtils.getFilenameWithoutExtension(item.fullPath);
+        FileSystem.resolve(basename + ".wast", (notFound) => {
+            if (notFound) {
+                const projectId = PreferencesManager.getViewState("projectId");
+                _nodeDomain.exec("wasm2wast", projectId, brackets.app.convertRelativePath(item.fullPath)).done(() => {
+                    ProjectManager.refreshFileTree();
+                }).fail((err) => {
+                    showErrorDialog(ExtensionStrings.ERROR_TRANSLATE_WASM);
+                    console.log(err);
```

Comment:
console.error ?

Reply:
below error conditaion doesn't have console.log. is this necessary ?

Reply:
No. It isn't necessary. I add it to check 'err' parameter.
Below conditon doesn't have any parameters so that I doesn't add console.log.